README.rst

```diff
@@ -65,21 +63,21 @@ Here is a quick example of plotting an AIA image:
 Getting Help
 ============
 
-For more information or to ask questions about SunPy, check out:
+For more information or to ask questions about ``sunpy`` or any other SunPy library, check out:
 
--  `SunPy Documentation`_
+-  `sunpy Documentation`_
 -  `SunPy Element Channel`_
 -  `SunPy Mailing List`_
 
-.. _SunPy Documentation: https://docs.sunpy.org/en/stable/
+.. _sunpy Documentation: https://docs.sunpy.org/en/stable/
 .. _SunPy Element Channel: https://app.element.io/#/room/#sunpy:openastronomy.org
 .. _SunPy Mailing List: https://groups.google.com/forum/#!forum/sunpy
 
-Acknowledging or Citing sunpy
-=============================
+Acknowledging or Citing ``sunpy``
+=================================
 
-If you use `sunpy` in your scientific work, we would appreciate citing it in your publications.
-The continued growth and development of `sunpy` is dependent on the community being aware of `sunpy`.
+If you use ``sunpy`` in your scientific work, we would appreciate citing it in your publications.
+The continued growth and development of `sunpy` is dependent on the community being aware of ``sunpy``.
 
 Please see https://sunpy.org/about#acknowledging-or-citing-sunpy on how to do this.
```

Comment:
A bit weird since that page is still chock full of not distinguishing between sunpy and SunPy

Reply:
One problem at a time :P